Library/src/Multiplayer/SpaceEntity.cpp

```diff
@@ -1066,7 +1066,14 @@ bool SpaceEntity::Deselect()
 
 bool SpaceEntity::IsModifiable()
 {
-	return (OwnerId == EntitySystem->GetMultiplayerConnection()->GetClientId() || IsTransferable);
+    if (EntitySystem != nullptr && EntitySystem->GetMultiplayerConnection() != nullptr)
+    {
+	    return (OwnerId == EntitySystem->GetMultiplayerConnection()->GetClientId() || IsTransferable);
+    }
+    else
+    {
+        return true;
```

Comment:
Suggestion - Perhaps a comment here to explain the rationale? That having no connection means that the local application has permission to mutate any state on any entity, whereas when connected the application can only mutate those entities which can be owned by the local user?